9c-main/multiplanetary/network/heimdall.yaml

```diff
@@ -0,0 +1,430 @@
+# Default values for 9c-main cluster.
+# This is a YAML-formatted file.
+
+clusterName: "9c-main-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: "longfin/9c-headless"
+    tag: "multiplanetary-2023111801"
```

Comment:
```suggestion
```
Remove this value to use the general image?

Reply:
Now, the general's image is the mainnet headless image, but we should set the image to `longfin/9c-headless:multiplanetary-2023111801` only in heimdall